tests/behat/features/workflow.feature

```diff
@@ -278,11 +278,6 @@ Feature: Workflow states and transitions
     And I should see the text "[TEST] Editor Test title"
     And I should see the text "Unpublished" in the "[TEST] Editor Test title" row
 
-    When I go to "admin/content/moderated"
-    And the response status code should be 200
-    And I should see the text "[TEST] Editor Test title"
-    And I should see the text "Needs Review" in the "[TEST] Editor Test title" row
-
```

Comment:
this test is not relevant for this module anymore as this module does not provide the moderated content filters view.